modules/runtime/pages/performance-troubleshooting.adoc

```diff
@@ -102,6 +102,28 @@ The execution threshold value can be configured in `bonita-platform-community.pr
 bonita.platform.persistence.dbquery.warnWhenLongerThanMillis=1000
 ----
 
+[#monitor-transaction-isolation]
+== Monitor Transaction isolation
+
+You may encounter the following log during execution:
+----
+"Within the same transaction, the Search count & page results are not consistent."
+----
+For your information, when Bonita performs a search to build a list of elements (e.g. the Bonita applications listing the process definitions, process instances, human tasks, ...) it always performs two queries on the DB:
+
+ - a first query to count the TOTAL amount of elements;
+ - a second query to actually retrieve the first PAGE_SIZE elements.
+
+If the TOTAL number of elements is smaller than the PAGE_SIZE, Bonita then compares the TOTAL count against the number of elements retrieved by the second query.
+Depending on the number of elements in the DB, those queries may take a long time to execute, thus some update may take place while performing the first query.
```

Comment:
This is not true. In this precise case, the problem is that the count query and the search query do no match exactly, so this is a dev problem. The query has to be fixed.